### ISSUE TITLE
Hyperparameters with UTAE model

### ISSUE THREAD

**Issue by PedroConrado** [VSainteuf/utae-paps#31]
Currently trying to reproduce the results of the segmentation experiments on the PASTIS benchmark using the UTAE model.
Were the default script value used (learning rate of 0.001 and batch size of 4), or different hyperparameters were applied?

**Comment by VSainteuf** [comment]
Hi @PedroConrado 
In the paper, the semantic segmentation experiments use a fixed learning rate of 0.001 for 100 epochs. For the panoptic segmentation experiments, we start with a higher learning rate of 0.01 for 50 epochs, and decrease it to 0.001 for the last 50 epochs (see appendix of the paper).
